docs/notebooks/utils/downloading_brains.ipynb

```diff
@@ -131,5 +110,6 @@
    "outputs": [],
    "source": []
   }
+
  ]
-}
```

Comment:
you deleted the entire downloading_brains notebook? Is this necessary for your PR to pass?